lib/modules/datasource/flutter/index.ts

```diff
@@ -0,0 +1,50 @@
+import { Datasource } from '../datasource';
+import type { GetReleasesConfig, ReleaseResult } from '../types';
+import type { FlutterRelease } from './types';
+
+export class FlutterDatasource extends Datasource {
+  static readonly id = 'flutter';
```

Comment:
```suggestion
  static readonly id = 'flutter-version';
```
Rename to `flutter-version` as @rarkins suggests